wicketd/src/update_tracker.rs

```diff
@@ -636,6 +638,65 @@ impl UpdateDriver {
                 )
                 .register();
 
+        // The SP only has one updateable firmware slot ("the inactive bank").
+        // We want to ask about slot 0 (the active slot)'s current version, and
+        // we are supposed to always pass 0 when updating.
+        let sp_firmware_slot = 0;
+
+        // To update the SP, we want to know both its version and its board (so
+        // we can map to the correct artifact from our update plan).
+        let sp_artifact_and_version = sp_registrar
+            .new_step(
+                UpdateStepId::InterrogateSp,
+                "Checking current SP version",
```

Comment:
Should we update this message to indicate we are also now checking the board revision?